### PR TITLE
Adjusted Homun S Quest to support Biolo

### PR DESCRIPTION
**Pull request by mazvi**
Homun S Quest Fixed Job Class for after Genetic

Thanks to @Lemongrass3110

`Co-authored-by: Lemongrass3110 <lemongrass@kstp.at>`

* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**:

### CODE REVIEW

File: npc/re/quests/homun_s.txt [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 //= 1.2 Replaced with official script. [Euphy]
+//= 1.3 Added Job Class Biolo. [Mazvi]
 //============================================================ 
 
 // Main NPCs :: Homun_S
 //============================================================
 job3_gen01,12,44,3	script	Viorel#job3_gen01	542,{
 	if (countitem(6415)) {
-		if (Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic) {
+		if (Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic  || Class == Job_Biolo) {
```
Comment (Lemongrass3110) [inline npc/re/quests/homun_s.txt:24]:
```suggestion
		if ((eaclass()&EAJ_THIRDMASK)==EAJ_GENETIC) {
```

File: npc/re/quests/homun_s.txt [hunk 2]
```diff
@@ -66,7 +67,7 @@ job3_gen01,12,44,3	script	Viorel#job3_gen01	542,{
 			close;
 		}
 	}
-	if ((Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic) && (checkquest(4154) > -1)) {
+	if ((Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic  || Class == Job_Biolo) && (checkquest(4154) > -1)) {
```
Comment (Lemongrass3110) [inline npc/re/quests/homun_s.txt:70]:
```suggestion
	if (((eaclass()&EAJ_THIRDMASK)==EAJ_GENETIC) && (checkquest(4154) > -1)) {
```

File: npc/re/quests/homun_s.txt [hunk 3]
```diff
@@ -130,7 +131,7 @@ job3_gen01,12,44,3	script	Viorel#job3_gen01	542,{
 		mes "[Viorel]";
 		mes "Oh. Of course you need to have the suitable potentials. You could probably prove yourself once you become a ^FF4800Genetic^000000.";
 		close;
-	} else if (Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic) {
+	} else if (Class == Job_Genetic || Class == Job_Genetic_T || Class == Job_Baby_Genetic  || Class == Job_Biolo) {
```
Comment (Lemongrass3110) [inline npc/re/quests/homun_s.txt:134]:
```suggestion
	} else if ((eaclass()&EAJ_THIRDMASK)==EAJ_GENETIC) {
```

**Review by Lemongrass3110 — COMMENTED** [review]
I think this way might be better.

**Comment by admkakaroto** [comment]
Not need change:
`} else if (Class == Job_Novice_High || Class == Job_Merchant_High || Class == Job_Alchemist || Class == Job_Creator) {`

to:
`} else if (Class == Job_Novice_High || Class == Job_Merchant_High || Class == Job_Alchemist || Class == Job_Creator || Class == Job_Biolo) {`

too?

**Comment by mazvi** [comment]
> Not need change :
> 
> LINE: https://github.com/rathena/rathena/pull/6634/files#diff-bb5b36f357a8a512de61611fc7a817a3a8934562c856c50fa103980bc59b5c44L45
> 
> `} else if (Class == Job_Novice_High || Class == Job_Merchant_High || Class == Job_Alchemist || Class == Job_Creator) {`
> 
> to: `} else if (Class == Job_Novice_High || Class == Job_Merchant_High || Class == Job_Alchemist || Class == Job_Creator || Class == Job_Biolo) {`
> 
> too?

I don't think it needs to be changed because it's only up to the job creator

**Pull request merged**